cluster-scope/base/core/namespaces/peribolos-as-a-service/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: peribolos-as-a-service
+    annotations:
+        openshift.io/display-name: Peribolos Service
+        openshift.io/requester: peribolos
+        op1st/project-owner: open-services-group
+        op1st/onboarding-issue: "https://github.com/open-services-group/peribolos-as-a-service/issues/31"
+        op1st/docs: "https://github.com/AICoE/meteor-operator"
```

Comment:
We can change the docs link to: https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos if we don't have anything else available.

Reply:
sorry, missed this. will update.